app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+  def new
+    # Renders the login form
+  end
```

Comment:
```suggestion
  def new; end
```
Viết gọn trên 1 line e nhé